### ISSUE TITLE
Upgrade "QUEUE SETTINGS" under Receive

### ISSUE THREAD

**Issue by Mrc0113** [SolaceLabs/solace-tryme-cli#20]
If I do a `stm receive -h` it's confusing that topic is under "QUEUE SETTINGS". 
I assume how this works is as follows: 
1. If i just have a topic, or list of topics then the app just subscribes to those topics. No queue involved. 
2. If I just have a queue, then it binds to the queue
3. If I have a queue and topics then it will create a queue (if missing) and add the topic subscriptions if I use the `create-if-missing`  flag? 
4. If I have a queue and topics then it will fail if the queue doesn't exist if I do NOT set `create-if-missing`
5. If i have a queue and topics then what is expected to happen if the queue is there but the queue doesn't have the right topic subscriptions? It just added the subscription to the queue for me but that felt unexpected? It feels like it should fail with an error? 

Suggested changes: 
1. The name of this section: "QUEUE" settings is confusing if I'm just using topic subscriptions. I think we need to change this section to "Subscriptions & Endpoints" or a better name if you have it? 
```
  /* QUEUE SETTINGS */
  -t, --topic <TOPIC...>                   the message topic(s) (default: ["solace/try/me"])
  -q, --queue <QUEUE>                      the message queue
```
2. We should add `create-if-missing` to this section
3. Because we have the concept of topic subscriptions and also topic endpoints we should make it clear that `-t` is for topic subscriptions. So I guess change it to "the message topic subscription(s)" 
4. At "endpoint" to the end of the "the message queue" description

**Comment by Mrc0113** [comment]
Specifying the topic is a subscription also applies to the `reply` command.

**Comment by gvensan** [comment]
Fixed in v0.0.36
